package/MDAnalysis/core/groups.py

```diff
@@ -1182,17 +1214,63 @@ def select_atoms(self, sel, *othersel, **selgroups):
                 therefore have atoms that were initially absent from the
                 instance :meth:`~select_atoms` was called from.
 
+            global *selection*
```

Comment:
Here _selection_ is indeed the kwarg name, and not a selection expression/keyword. I'll put it in backticks for correctness but am not sure that having it formatted as such in the docs helps much, since the user has nowhere to know what that kwarg is (it's only explained in the doc  body of the `global` keyword).